regressionTests/testjs/src/groups/addressGroup.ts

```diff
@@ -0,0 +1,43 @@
+import {TestSuite} from "../resultClasses";
+import {AddressV1_0Handler} from "../handlers/v1.0/addressV1_0Handler";
+
+/*
+    AddressGroup handles the test for the endpoints of the address group
+ */
+export class AddressGroup {
+    v1_0Handler: AddressV1_0Handler;
+
+    constructor(v1_0Handler: AddressV1_0Handler) {
+        this.v1_0Handler = v1_0Handler;
+    }
+
+    async handleAddressShouldWork(): Promise<Array<TestSuite>> {
+        let v1_0result = await this.v1_0Handler.handleAddressShouldWork();
```

Comment:
And if we would have more handlers, for more versions, we will call all of them here, right?

Reply:
That's true. When we will have multiple versions, we can have a `multi-select` and based on the selected values we will call handlers for them